.env

```
@@ -0,0 +1,3 @@
+RABBITMQ_ERLANG_COOKIE=rabbitmqCookie
+RABBITMQ_DEFAULT_USER=admin
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The RABBITMQ_DEFAULT_USER key should go before the RABBITMQ_ERLANG_COOKIE key

(UnorderedKey)